### ISSUE TITLE
'UIApplicationOpenURLOptionsSourceApplicationKey' error compiling

### ISSUE THREAD

**Issue by AaronZhouQian** [braintree/braintree_ios#358]
### General information

* SDK/Library version: Braintree (4.9.2)
* Environment: N.A.
* iOS Version and Device: iOS 11, Xcode 9.0 (9A235). 
* Integration type and version:  Example: CocoaPods 1.3.1

### Issue description

After installing the Braintree pod, using `pod instal`, Xcode cannot build the framework. Specifically, inside BTAppSwitch.m line 40

```
#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 90000
+ (BOOL)handleOpenURL:(NSURL *)url options:(NSDictionary *)options {
    return [[[self class] sharedInstance] handleOpenURL:url sourceApplication:options[UIApplicationOpenURLOptionsSourceApplicationKey]];
}
#else
+ (BOOL)handleOpenURL:(NSURL *)url options:(__unused NSDictionary *)options {
    return [[[self class] sharedInstance] handleOpenURL:url sourceApplication:nil];
}
#endif
```
The compiler says:
```'UIApplicationOpenURLOptionsSourceApplicationKey' is only available on iOS 9.0 or newer
Enclose 'UIApplicationOpenURLOptionsSourceApplicationKey' in an @available check to silence this warning

```

**Comment by bluk** [comment]
I think `pod repo update` and `pod update Braintree` should fix this with the 4.9.3 release. For details, please see https://github.com/braintree/braintree_ios/issues/353 .

**Comment by bluk** [comment]
If this issue isn't resolved with `4.9.3`, please leave a comment and we can re-open or open a new issue. Thanks.
